Code/Tools/SerializeContextTools/Runner.cpp

```diff
@@ -93,6 +93,26 @@ namespace SerializeContextTools
         AZ_Printf("Help", R"(    example: 'createtype --type-name="AZ::Entity" --json-prefix="/My/Anchor"')" "\n");
         AZ_Printf("Help", R"(    example: 'createtype --type-name="AZ::Entity" --output-file=object.json)" "\n");
         AZ_Printf("Help", "\n");
+        AZ_Printf("Help", "  'createuuid': Create a UUID using a SHA1 hash from a string and output the contents to stdout or a file.\n");
+        AZ_Printf("Help", "    [arg] --values=<string...>: One or more strings to convert to UUID.\n");
```

Comment:
Nit: Could also mention the delimiter in the help string

Reply:
Actually this is part of the CommandLine API.
Specifying multiple `--values` arguments or using a comma is the way to separate multiple of an [arugment](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Settings/CommandLine.h#L20-L30)